doc/classes/Transform2D.xml

```diff
@@ -78,6 +78,13 @@
 				This method does not account for translation (the origin vector).
 			</description>
 		</method>
+		<method name="determinant" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the determinant of the basis matrix. If the basis is uniformly scaled, its determinant is the square of the scale.
+				A negative determinant means the basis has a negative scale. A zero determinant means the basis isn't invertible, and is usually considered invalid.
```

Comment:
The first sentence: what does "a negative scale" is meant to mean? Because for scale like `(-1, -1)` the determinant would be positive. Negative determinant means the scale's components has opposite signs (and both are non-zero).

Reply:
Technically a scale of (-1, -1) can't be decomposed because it would be interpreted as a 180 degree rotation instead. However, indeed we can be more clear here, I'll update the text.